tests/testthat/test-plumber.R

```diff
@@ -37,6 +37,21 @@ test_that("Invalid file fails gracefully", {
   expect_error(plumber$new("asdfsadf"), regexp="File does not exist.*asdfsadf")
 })
 
+test_that("#plumb accepts a directory with a `plumber.R` file", {
+  # works without trailing slash
+  r <- plumb(dir = 'files')
+  expect_equal(length(r$endpoints), 1)
```

Comment:
You don't need to care about the internal details, but this test doesn't actually mean much about what was found in the file. Can we add an additional test after this (and line 45) for:

`expect_equal(length(r$endpoints[[1]]), 4)`

to check the contents of this length-1 list?